src/governance.ts

```diff
@@ -136,15 +136,15 @@ export class GovernanceCanister {
    */
   public getProposalInfo = async ({
     proposalId,
-    certified = true
+    certified = true,
   }: {
-    proposalId: bigint,
-    certified?: boolean
+    proposalId: bigint;
+    certified?: boolean;
   }): Promise<ProposalInfo | null> => {
     const service = certified ? this.certifiedService : this.service;
```

Comment:
can you extract a function like `getGovernanceService` or add a TODO, seems this code is already duplicated